cmd/pulumi-test-language/test_host.go

```diff
@@ -83,14 +83,32 @@ func (h *testHost) ListAnalyzers() []plugin.Analyzer {
 }
 
 func (h *testHost) Provider(descriptor workspace.PackageDescriptor) (plugin.Provider, error) {
-	// Look in the providers map for this provider
+	// If we've not been given a version, we'll try and find the provider by name alone, erroring if there are multiple
```

Comment:
Normal behavior is to return the latest, not error. Won't effect tests today, but probably worth changing.